packages/babel-core/test/config-chain.js

```diff
@@ -294,4 +294,178 @@ describe("buildConfigChain", function () {
 
     assert.deepEqual(chain, expected);
   });
+
+  it("js-config", function () {
+    const chain = buildConfigChain({
+      filename: fixture("js-config", "src.js")
+    });
+
+    const expected = [
+      {
+        options: {
+          plugins: [
+            "foo",
+            "bar"
+          ]
+        },
+        alias: fixture("js-config", ".babelrc.js"),
+        loc: fixture("js-config", ".babelrc.js"),
+        dirname: fixture("js-config")
+      },
+      {
+        options: {
+          ignore: [
+            "root-ignore"
+          ]
+        },
+        alias: fixture(".babelignore"),
+        loc: fixture(".babelignore"),
+        dirname: fixture()
+      },
+      {
+        options: {
+          filename: fixture("js-config", "src.js"),
+        },
+        alias: "base",
+        loc: "base",
+        dirname: fixture("js-config")
+      }
+    ];
+
+    assert.deepEqual(chain, expected);
+  });
+
+  it("js-config-default", function () {
```

Comment:
This test is currently failing because of the `__esModule` check. Will update when we figure out what we want to do with that.